Dashboard/app/js/templates/navSurveys/nav-surveys-main.handlebars

```diff
@@ -50,7 +50,7 @@
                 <li><a class="addSurvey noChanges">{{t _create_new_survey}}</a></li>
               {{else}}
                 {{#if view.disableAddSurveyButtonInRoot}}
-                {{!--  {{t _survey_only_in_folder}} --}}
+                <!--  {{t _survey_only_in_folder}} -->
```

Comment:
If not in use, let's remove it

Reply:
Its in use. but because I pass it in as a key below, the translation parser doesnt recognize it. I'll add comment to clarify that.